### DISCUSSION TITLE
Dynamic routes on vercel production

### DISCUSSION THREAD

**Discussion by BBartt** [vercel/next.js#55659] [App Router]
Hi,

I am working on next v13 with app folder.
I have created dynamic routes `app/test/[params]/page.tsx` and I got 404 error page when I access this route.
I added `export const dynamicParams = true;` line to my `...[params]/page.tsx` but I still I see 404 page on vercel.

My next.config.js file is:
`
const nextConfig = {
  output: "export",
  distDir: "_static",
  images: {
    unoptimized: true,
  },
  trailingSlash: true,
  experimental: {
    appDir: true,
  },
};`

How can make dynamic route work on production?

**Comment by bananaNuggets75** [comment]
did you fix it? I am having quite a similar problem, but mine does not give 404 error just redirects back to the homepage and gives me the correct url
